pages/algorithms/equivalent-purpose.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Equivalent purpose 
+key: equivalent-purpose 
+---
+
+Resources that are not identical can still serve an equivalent purpose by both/all living up to the expectation that was created when sending the user to the resources. This would usually involve that the advertised key content is the same.  
```

Comment:
I don't think this definition is as clear as it can be. Here is my suggestion (see if you think it improves):
Not identical resources can still serve an equivalent purpose by equally complying to the expectation formed by the user when navigating to them.

Reply:
It should be "Non-identical resources"